modules/services/twmn.nix

```
@@ -0,0 +1,347 @@
+# twmn service
+{ config, lib, pkgs, stdenv, ... }:
+with lib;
+let
+  cfg = config.services.twmn;
+
+  animationOpts = {
+    curve = mkOption {
+      type = types.ints.between 0 40;
+      default = 38;
+      example = 19;
+      description = ''
+        the qt easing-curve animation to use for the animation, See
+        <link href="https://doc.qt.io/qt-5/qeasingcurve.html#Type-enum">
+        QEasingCurve documentation</link>.
+      '';
+    };
+    duration = mkOption {
+      type = types.ints.unsigned;
+      default = 1000;
+      example = 618;
+      description = "the animation duration in milliseconds.";
+    };
+  };
+in {
+  meta.maintainers = with maintainers; [ austreelis ];
+
+  options.services.twmn = {
+    duration = mkOption {
+      type = types.ints.unsigned;
+      default = 3000;
+      example = 5000;
+      description = ''
+        the time each notification remains visible, in milliseconds.
+      '';
+    };
+    enable = mkEnableOption "tiling window manager notification daemon";
```

please put this as the first option

```suggestion
    enable = mkEnableOption "twmn, a tiling window manager notification daemon";
```